docs/GraFx-Developers/connectors/authorization-for-connectors/index.md

```
@@ -164,23 +164,23 @@ In addition to specifying the authorization type using the `-at` argument, you m
 2. The `grant_type` for OAuth 2.0 types is automatically set as per the specification and cannot be configured.
-3. `access_token` and `refresh_token` are cached internally for OAuth 2.0 types. For OAuth 2.0 Authorization Code you can clear your own cached session — see [Signing out of an OAuth 2.0 Authorization Code session](#signing-out-of-an-oauth-20-authorization-code-session). For the other OAuth 2.0 types there is no way to reset them without deleting the Connector.
+3. `access_token` and `refresh_token` are cached internally for OAuth 2.0 types. For OAuth 2.0 Authorization Code you can clear your own cached session — see [Signing out of an OAuth 2.0 Authorization Code session](#signing-out-of-an-oauth-20-authorization-code-session). For the other OAuth 2.0 types there is no way to reset them without deleting the connector.
 4. OAuth 2.0 Authorization Code is user-specific and not useful with an Integration User (used outside CHILI GraFx).
 5. OAuth 2.0 Authorization Code is limited to "browser" usage, which may restrict its applicability in certain scenarios.
 
 #### Signing Out of an OAuth 2.0 Authorization Code Session
 
-Because OAuth 2.0 Authorization Code authorizes each user individually, every user builds up their own authorization session for a Connector. To end your own session, call:
+Because OAuth 2.0 Authorization Code authorizes each user individually, every user builds up their own authorization session for a connector. To end your own session, call:
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)